resources/build/help-keyman-com.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+#
+# Prevents 'clear' on exit of mingw64 bash shell
+#
+SHLVL=0
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+#
+# Allows us to check for existence of subfolders in help/
+#
+shopt -s nullglob
+
+#
+# In this script we update the git repo and copy the Keyman for [Platform]
+# documentation over. This should be run after a full build.
+#
+
+#
+# These are passed via environment:
+#
+# HELP_KEYMAN_COM = the home of the help.keyman.com repository
+# PLATFORM = platform of the Keyman help to upload
+#
+# That repo must have push to origin configured and logged in
+#
+
+if [ -z ${HELP_KEYMAN_COM+x} ]; then
+  >&2 echo "Not uploading documentation: must set HELP_KEYMAN_COM in environment."
+  exit 1
+fi
+
+if [ -z ${PLATFORM} ]; then
+  >&2 echo "No uploading documentation: must set PLATFORM in environment."
+  exit 1
+fi
+
+if [ ! -d "$HELP_KEYMAN_COM/products/" ]; then
+  >&2 echo "HELP_KEYMAN_COM path ($HELP_KEYMAN_COM) does not appear to be valid."
+  exit 1
+fi
+
+#
+# Environment
+#
+
+echo "Uploading Keyman for $PLATFORM documentation to help.keyman.com"
+
+#
+# Uploading Keyman documentation
+#
+
+##
+## Upload documentation updates to help.keyman.com
+## Paths depend on $PLATFORM
+##
+function upload_keyman_help {
+
+  local helppath
+  local dstpath
+
+  case $PLATFORM in
+    android)
+      helppath=$KEYMAN_ROOT/android/help
+      dstpath="$HELP_KEYMAN_COM/products/android/$VERSION_RELEASE"
+      ;;
+    ios)
+      helppath=$KEYMAN_ROOT/ios/help
+      dstpath="$HELP_KEYMAN_COM/products/iphone-and-ipad/$VERSION_RELEASE"
+      ;;
+    linux)
+      helppath=$KEYMAN_ROOT/linux/help
+      dstpath="$HELP_KEYMAN_COM/products/linux/$VERSION_RELEASE"
+      ;;
+    mac)
+      helppath=$KEYMAN_ROOT/mac/help
+      dstpath="$HELP_KEYMAN_COM/products/mac/$VERSION_RELEASE"
+      ;;
+    windows)
+      helppath=$KEYMAN_ROOT/windows/bin/help/md/desktop
```

Comment:
🤣 love how the Windows help is so convoluted in comparison. Might be worth adding a comment here to note that the Windows build must have completed before this script can be run.